portfolio/src/main/webapp/script.js

```diff
@@ -12,32 +12,64 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-/**
- * Adds a random quote to the page.
- */
-function addRandomQuote() {
-const quotes =
-      ['\"There\'s only one thing I hate more than lying: skim milk. Which is water lying about being milk.\" - Ron Swanson', 
-      '\"Just remember, every time you look up at the moon, I, too, will be looking at a moon. Not the same one, obviously. That\'s impossible.\" - Andy Dwyer',
-      '\"I\'ve made a huge mistake.\" - Gob Bluth', '\"There\'s always money in the banana stand.\" - George Bluth',
-      '\"Would I rather be feared or loved? Easy. Both. I want people to be afraid of how much they love me.\" - Michael Scott',
-       '\"Kids, you tried your best and you failed miserably. The lesson is never try.\" - Homer Simpson'];
-
-  // Pick a random quote.
-  const quote = quotes[Math.floor(Math.random() * quotes.length)];
-
-  // Add it to the page.
-  const quoteContainer = document.getElementById('quote-container');
-  quoteContainer.innerText = quote;
-}
+// /**
+//  * Adds a random quote to the page.
+//  */
+// function addRandomQuote() {
+// const quotes =
+//       ['\"There\'s only one thing I hate more than lying: skim milk. Which is water lying about being milk.\" - Ron Swanson', 
+//       '\"Just remember, every time you look up at the moon, I, too, will be looking at a moon. Not the same one, obviously. That\'s impossible.\" - Andy Dwyer',
+//       '\"I\'ve made a huge mistake.\" - Gob Bluth', '\"There\'s always money in the banana stand.\" - George Bluth',
+//       '\"Would I rather be feared or loved? Easy. Both. I want people to be afraid of how much they love me.\" - Michael Scott',
+//        '\"Kids, you tried your best and you failed miserably. The lesson is never try.\" - Homer Simpson'];
+
+//   // Pick a random quote.
+//   const quote = quotes[Math.floor(Math.random() * quotes.length)];
+
+//   // Add it to the page.
+//   const quoteContainer = document.getElementById('quote-container');
+//   quoteContainer.innerText = quote;
+// }
+
+// /**
+//  * Adds a random quote to the page using fetch() to request content from the server.
+//  */
+// function getRandomQuoteUsingArrowFunctions() {
+//   fetch('/data').then(response => response.text()).then((quote) => {
+//     console.log('Adding quote to dom: ' + quote);
+//     document.getElementById('quote-container').innerText = quote;
+//   });
+// }
```

Comment:
why commiting this commented code?

Reply:
I commented this code in case if I need it for future steps in this project, but come to think of it I don't foresee that being the case (and even it does I could revert to an older commit/branch that still has this code). I'm making a new commit now that removes these commented functions.